From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ryder Belserion <no-reply@ryderbelserion.com>
Date: Fri, 13 Jan 2023 03:16:40 -0500
Subject: [PATCH] Fix version checker


diff --git a/src/main/java/com/destroystokyo/paper/PaperVersionFetcher.java b/src/main/java/com/destroystokyo/paper/PaperVersionFetcher.java
index c43e644e36fe4a39b48f59a47ee8f1022a7f1ecd..43e74fe8108197a4490efee036616683a5c9da1e 100644
--- a/src/main/java/com/destroystokyo/paper/PaperVersionFetcher.java
+++ b/src/main/java/com/destroystokyo/paper/PaperVersionFetcher.java
@@ -33,7 +33,7 @@ public class PaperVersionFetcher implements VersionFetcher {
     @Nonnull
     @Override
     public Component getVersionMessage(@Nonnull String serverVersion) {
-        String[] parts = serverVersion.substring("git-Tentacles-".length()).split("[-\\s]"); // Purpur // Tentacles
+        String[] parts = serverVersion.substring("git-Quill-".length()).split("[-\\s]"); // Purpur // Quill
         final Component updateMessage = getUpdateStatusMessage("Parchat/Quill", "ver/" + getMinecraftVersion(), parts[0]); // Purpur // Quill
         final Component history = getHistory();
 
